code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -730,4 +730,16 @@ var/global/list/animal_count = list() //Stores types, and amount of animals of t
 /mob/living/simple_animal/proc/pointed_at(var/mob/pointer)
 	return
 
+/mob/living/simple_animal/proc/space_check() //Returns a 1 if you can move in space or can kick off of something, 0 otherwise
+	if(Process_Spacemove())
+		return 1
+	var/spaced = 1
+	for(var/turf/T in range(src,1))
+		if(!istype(T, /turf/space))
+			spaced = 0
```

Comment:
Fair point, figured Process_Spacemove'd catch that, or it was a turf type.

The break's a good call, too